packages/dapp/public/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Smart Invoice",
   "short_name": "Smart Invoice",
-  "description": "Smart Invoice",
+  "description": "Smart Invoice is an easy-to-use tool that provides web3 freelancers with cryptocurrency invoicing, escrow, and arbitration.",
```

Comment:
If Gnosis Safe uses this field as the tagline on the Safe App list, it might work to add 'customizable, 'flexible', or a similar adjective in front of cryptocurrency invoicing to sell the flexibility aspect of it. And overall any language to broaden the appeal. So a potential option could be:

"Smart Invoice is an easy-to-use tool that provides freelancers and contractors with customizable cryptocurrency invoicing, escrow, and arbitration."

Reply:
I just copied from the website, I like flexible better than customizable. The main point was to expand the description for how it is displayed on the safe apps page.

Reply:
I'm going to merge for now, but feel free to update any language here.